h/api/logic.py

```diff
@@ -12,6 +12,17 @@
 PROTECTED_FIELDS = ['created', 'updated', 'user', 'consumer', 'id']
 
 
+def search_annotations(params, user=None, search_normalized_uris=False):
+    results = search_lib.search(request_params=params,
+                                user=user,
+                                search_normalized_uris=search_normalized_uris)
+
+    return {
+        'total': results['total'],
+        'rows': [search_lib.render(a) for a in results['rows']],
+    }
+
+
```

Comment:
The feed code isn't actually using this anymore, but we may as well leave it in because it matches the pattern that the rest of the API views on master use currently